chapters/book/modules/chapter_2/pages/event.adoc

```diff
@@ -6,9 +6,18 @@ Events are a useful tool for logging and notifying external entities about speci
 
 == Defining Events
 
-To define an event in a Starknet contract, you can use the **`#[event]`** attribute followed by the event definition. In our voting contract, we have two events, **`VoteCast`** and **`UnauthorizedAttempt`**.
+To define an event in a Starknet contract, you can use the **`#[event]`** attribute followed by defining an **`enum`** that will contain the event variants. 
```

Comment:
```suggestion
To define an event in a Starknet contract, you can use the **`#[event]`** attribute followed by defining an **`enum`** that contains the event variants. 
```

Reply:
@OkoliEvans I don't see that my comments were implemented. 

Just to be clear, the changes I requested are based on our style guide and general writing guidelines.

For more information, see the [Starknet documentation supplementary style guide](https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/starknet_docs_style_guide.adoc). and [Contributing to Starknet documentation](https://github.com/starknet-io/starknet-docs#contributing-to-starknet-documentation).